File: Solutions/1347. Minimum Number of Steps to Make Two Strings Anagram/1347. Minimum Number of Steps to Make Two Strings Anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int res=0;
        int[] cnt=new int[26];
        for(var c:s.toCharArray())
            ++cnt[c-'a'];
        for(var c:t.toCharArray())
            --cnt[c-'a'];
        for(int i=0;i<26;++i)
            res+=(cnt[i]==0)?0:Math.abs(cnt[i]);
        return res/2;
    }
}

File: Solutions/1347. Minimum Number of Steps to Make Two Strings Anagram/1347. Minimum Number of Steps to Make Two Strings Anagram.go
func minSteps(s string, t string) int {
    res,cnt:=0,make([]int,26)
    for _,c:=range(s){
        cnt[c-'a']+=1
    }
    for _,c:=range(t){
        cnt[c-'a']-=1
    }
    for i:=0;i<26;i+=1{
        if(cnt[i]==0){
            res+=0
        }else{
            res+=abs(cnt[i])
        }
    }
    return res/2
}
func abs(x int)int{
    if(x<0){
        return -x
    }
    return x
}